lib/client.js

```diff
@@ -144,6 +144,13 @@ var warmupLimit = 32;
  * @property {Number} [fetchSize] Amount of rows to retrieve per page.
  * @property {Array|Array<Array>} [hints] Type hints for parameters given in the query, ordered as for the parameters.
  * <p>For batch queries, an array of such arrays, ordered as with the queries in the batch.</p>
+ * @property {Boolean} [isIdempotent] Defines whether the query can be applied multiple times without changing the result
+ * beyond the initial application.
+ * <p>
+ *   The query execution idempotence can be used at [RetryPolicy]{@link module:policies/retry~RetryPolicy} level to
+ *   determine if an statement can be retried in case of replica unavailability or write timeout.
```

Comment:
Shouldn't this be

> ...determine if a statement can be retried in case of request error or write timeout.

Unavailability always defers to child policy, which request error checks idempotence like write timeout.